hbase/src/main/java/org/apache/zeppelin/hbase/HbaseInterpreter.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.hbase;
+
+import org.apache.commons.exec.*;
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterPropertyBuilder;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.scheduler.Scheduler;
+import org.apache.zeppelin.scheduler.SchedulerFactory;
+import org.jruby.embed.LocalContextScope;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.jruby.embed.ScriptingContainer;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Created by rajatv on 4/22/15.
+ */
+public class HbaseInterpreter extends Interpreter {
+  Logger logger = LoggerFactory.getLogger(HbaseInterpreter.class);
+  int commandTimeOut = 600000;
+  ScriptingContainer scriptingContainer;
+
+  StringWriter writer;
+
+  static {
+    Interpreter.register("hbase", "hbase", HbaseInterpreter.class.getName(),
+        new InterpreterPropertyBuilder()
+            .add("hbase.home", "/usr/lib/hbase/", "Installation dir. of Hbase")
+            .add("hbase.ruby.sources", "lib/ruby",
+                "Path to Ruby scripts relative to 'hbase.home'")
+            .add("hbase.irb.load", "true", "Load hirb. Optional for testing only")
+            .build());
+  }
+
+  public HbaseInterpreter(Properties property) {
+    super(property);
+  }
+
+  @Override
+  public void open() {
+    String hbase_home = getProperty("hbase.home");
+    String ruby_src = getProperty("hbase.ruby.sources");
+    String abs_ruby_src = hbase_home + ruby_src;
```

Comment:
It will be good to log an error message with good explanation when some of the expected system properties is not provided.